app/controllers/atmosphere/api/v1/clew_controller.rb

```diff
@@ -23,7 +23,8 @@ def appliance_instances
 
         def appliance_types
           appliance_types = @appliance_types.active.includes(:tenants).references(:tenants).
-              includes(:appliance_configuration_templates).references(:appliance_configuration_templates).order(:id)
+              includes(:appliance_configuration_templates).references(:appliance_configuration_templates).
```

Comment:
Line is too long. [106/80]<br>Align the operands of an expression in an assignment spanning multiple lines.